beesdoo_product/__manifest__.py

```diff
@@ -10,30 +10,32 @@
 {
     "name": "beesdoo_product",
     "summary": """
-        Modification of product module for the needs of beescoop
+        Emptied.
         """,
     "author": "BEES coop - Cellule IT, Coop IT Easy SC",
     "website": "https://github.com/beescoop/Obeesdoo",
     "category": "Sales",
-    "version": "12.0.2.0.0",
+    "version": "12.0.3.0.0",
     "depends": [
         "product",
         "sale",
         "point_of_sale",
         "purchase",
+        "sale_product_deposit",
+        "product_scale_label",
+        "product_sale_limit_date",
+        "product_expiration",
+        "product_ingredients",
+        "product_hazard",
         "product_main_supplier",
         "sale_suggested_price",
         "sale_adapt_price_wizard",
+        "product_barcode_generator",
+        "product_label_print_request",
+        "beesdoo_product_label",
     ],
     "data": [
-        "data/product_label.xml",
-        "data/product_hazard.xml",
-        "data/barcode_rule.xml",
-        "data/product_sequence.xml",
         "views/beesdoo_product.xml",
```

Comment:
If it's emptied, why is this still here?

Reply:
It's leftover that we don't know where to put. After a talk with robin, we decided to leave them here, and remove them at next migration. I left a note in the description.